### PR TITLE
Aresina patch 1

### PR DESCRIPTION
**Pull request by aresina**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/aresina/teste/pull/6" target="_blank">ReviewNinja

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/aresina/teste/pull/6" target="_blank">ReviewNinja

File: README [hunk 1]
```diff
@@ -1,4 +1,6 @@
 
+
+
```
Comment (aresina) [inline README]: espaçamentos

File: src/init/Inicio.java [hunk 2]
```diff
@@ -6,6 +6,8 @@ public static void main(String[] args) {
 		Teste tes = new Teste();
 		
 		tes.Alerta();
+		
+		tes=null;
```
Comment (aresina) [inline src/init/Inicio.java]: !fixed

**Pull request merged**